app/models/resource.rb

```diff
@@ -1,6 +1,6 @@
 class Resource < ApplicationRecord
   include Elasticsearch::Model
-  index_name "articles-#{Rails.env}"
+  index_name "#{self.class.name.downcase}-#{Rails.env}"
```

Comment:
Actually, I've extracted this to `SearchIndex` in 8f8b584 also - this will be useful elsewhere.